pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -3445,6 +3449,92 @@ private void internalExpireMessagesByTimestampForSinglePartition(String subName,
         }
     }
 
+    private CompletableFuture<Void> internalExpireMessagesByTimestampForSinglePartitionAsync(String subName,
```

Comment:
conflict with  #13880

Reply:
comment in <https://github.com/apache/pulsar/pull/13880/files>

Reply:
Both PR's modify this method，warning: There may be conflict.

Reply:
Oh, I think we'd better close the new one and apply the fix from the old one.

Reply:
Ok,  I can rebase the code after https://github.com/apache/pulsar/pull/13880 merging to master to fix the conflict